packages/asc-ui/src/components/Tabs/TabList.tsx

```diff
@@ -1,45 +1,26 @@
-import React, {
-  HTMLAttributes,
-  forwardRef,
-  useState,
-  useEffect,
-  useImperativeHandle,
-  useRef,
-} from 'react'
-import TabListStyle, { TabListWrapper, TabListFade } from './TabListStyle'
+import styled from 'styled-components'
+import { themeColor, themeSpacing } from '../../utils'
 
-export default forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
-  ({ children, ...otherProps }, externalRef) => {
-    const [canScroll, setCanScroll] = useState<boolean>(false)
```

Comment:
Without checking if TabList can scroll you always have a small fade on the right, which you can see over the inset box shadow. It's quite small, but it can imply for some users that TabList is scrollable when it isn't. Should we keep canScroll here?